api/capi/src/nnstreamer-capi-single.c

```diff
@@ -997,7 +1034,8 @@ ml_single_invoke (ml_single_h single,
   }
 
   if (single_h->ignore_output == FALSE) {
-    *output = single_h->output;
+    if (need_alloc)
```

Comment:
At Line 1029. ```__invoke (single_h) ``` calls ```ml_tensors_data_clone_no_alloc ()```, which does ```single_h->output = NULL;```. Deleting the given output data. You may need to fix it before merging this for no-alloc.

Reply:
I added new flag about allocation.